sdk/translation/ai-translation-text/README.md

```diff
@@ -0,0 +1,324 @@
+
+With the value of the `TranslatorCredential` you can create the [TextTranslationClient][translator_client_class]:
+
+```typescript
+const translateCedential = new TranslatorCredential(apiKey, region);
+const translationClient = TextTranslationFactory(endpoint, translateCedential);
+```
+
+## Examples
+
+The following section provides several code snippets using the `client` [created above](#create-a-texttranslationclient-using-an-api-key-and-region-credential), and covers the main features present in this client library.
+
+### Get Supported Languages
+
+Gets the set of languages currently supported by other operations of the Translator.
+
+```typescript
+const langResponse = await translationClient.path("/languages").get();
+
+if (langResponse.status !== "200") {
+  const error = langResponse.body as ErrorResponseOutput;
+  throw error.error;
+}
+
+const languages = langResponse.body as GetLanguagesResultOutput;
+
+if (languages.translation) {
+  console.log("Translated languages:");
+  for (const key of languages.translation) {
+    const translationLanguage = languages.translation[key];
+    console.log(`${key} -- name: ${translationLanguage.name} (${translationLanguage.nativeName})`);
+  }
+}
+
+if (languages.transliteration) {
+  console.log("Transliteration languages:");
+  for (const key in languages.transliteration) {
+    const transliterationLanguage = languages.transliteration[key];
+    console.log(
+      `${key} -- name: ${transliterationLanguage.name} (${transliterationLanguage.nativeName})`
+    );
+  }
+}
+
+if (languages.dictionary) {
+  console.log("Dictionary languages:");
+  for (const key in languages.dictionary) {
+    const dictionaryLanguage = languages.dictionary[key];
+    console.log(
+      `${key} -- name: ${dictionaryLanguage.name} (${dictionaryLanguage.nativeName}), supported target languages count: ${dictionaryLanguage.translations.length}`
+    );
+  }
+}
+```
+
+Please refer to the service documentation for a conceptual discussion of [languages][languages_doc].
+
+### Translate
+
+Renders single source-language text to multiple target-language texts with a single request.
+
+```typescript
+const inputText: InputTextItem[] = [{ text: "This is a test." }];
+const parameters: TranslateQueryParamProperties & Record<string, unknown> = {
+  to: "cs",
+  from: "en",
+};
+const translateResponse = await translationClient.path("/translate").post({
+  body: inputText,
+  queryParameters: parameters,
+});
+
+if (translateResponse.status !== "200") {
+  const error = translateResponse.body as ErrorResponseOutput;
+  throw error.error;
+}
+
+const translations = translateResponse.body as TranslatedTextItemOutput[];
+for (const key in translations) {
+  const translation = translations[key];
+  console.log(
+    `Text was translated to: '${translation?.translations[0]?.to}' and the result is: '${translation?.translations[0]?.text}'.`
+  );
+}
```

Comment:
Could we loop on translations directly?
```suggestion
for (const translation of translations) {
  console.log(
    `Text was translated to: '${translation?.translations[0]?.to}' and the result is: '${translation?.translations[0]?.text}'.`
  );
}
```

Also, could we apply this change everywhere?

Reply:
Sure. Done